client/local/internal/internal.go

```diff
@@ -13,6 +13,17 @@ import (
 	"github.com/odpf/optimus/client/local"
 )
 
+func PathExist(specFS afero.Fs, path string) (bool, error) {
+	_, err := specFS.Stat(path)
+	if err == nil {
+		return true, nil
+	}
+	if os.IsNotExist(err) {
```

Comment:
On this, please use `errors.Is(...)` as this one is recommended by Go [here](https://pkg.go.dev/os#IsNotExist).